service/tutorial/order/OrderServices.xml

```diff
@@ -0,0 +1,360 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<services xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+          xsi:noNamespaceSchemaLocation="http://moqui.org/xsd/service-definition-2.1.xsd">
+
+    <!-- Order & Related Services -->
+    <service verb="create" noun="CreateOrder" displayName="Create the order">
```

Comment:
noun can only be "Order" here, the verb should tell the action, no need to add it again in the noun.

Reply:
Done.